.github/check_license.sh

```diff
@@ -14,9 +14,7 @@ echo "==========================="
 echo
 
 ERROR_FILES=""
-FILES_TO_CHECK=`find . \
-    -type f \( -name '*.sh' -o -name '*.py' -o -name 'Makefile' \) \
-    \( -not -path "*/.*/*" -not -path "*/third_party/*" \)`
+FILES_TO_CHECK=`git ls-files | grep -ve '^third_party\|^\.' | grep -e '\.sh$\|\.py$\|\.cpp$\|\.h$\|CMakeLists.txt$\|Makefile$'`
```

Comment:
Why did this change?  The "common" action is here: https://github.com/SymbiFlow/actions/blob/main/checks/check_python_script.sh

Reply:
That one has some problems. It fails on generated source (unless you also generate copyright headers, which is unnecessary), and also tries to check `.tox`. There is no reason to check anything that isn't checked in.